### ISSUE TITLE
Doesn't support file:// protocol

### ISSUE THREAD

**Issue by djmattyg007** [falconindy/pkgfile#23]
I run a local repository with packages on my computer. The tool I use creates a file for use by pkgfile, but pkgfile doesn't seem to want to download it. The only reason I can see for this is that it doesn't like file:// URLs.

**Comment by falconindy** [comment]
Shouldn't be a problem since it's just passed through to curl. Note that the URI actually will contain three slashes since you want to start at root, i.e. file:///path/to/some.db

Does pacman have any problem with this?

**Comment by falconindy** [comment]
Seems to work for me. I modified (part of) my pacman.conf to look like this:

    [falconindy]
    Server = file:///home/noclaf
    # Server = http://repo.falconindy.com/$repo/os/$arch

And then copied the .db and .files tarballs into /home/noclaf. pkgfile slurps it up just fine:

     # pkgfile -u
    :: Updating 10 repos...
      download complete: falconindy           [    35.9 KiB  10.5M/s  9 remaining]
      staging is up to date
      multilib-testing is up to date
      testing is up to date
      community-staging is up to date
      core is up to date
      extra is up to date
      multilib is up to date
      community is up to date
      community-testing is up to date
    :: download complete in 0.72s             <    35.9 KiB  49.9K/s  1 file     >

**Comment by djmattyg007** [comment]
I had this set for my custom repository:

    SigLevel = Required DatabaseRequired

When I removed it, pkgfile downloaded the files appropriately. When I put it back, it continued to download the files appropriately. I'm not really sure what changed, all I know is that I've had that line above since I added the custom repository.

It all appears to be working properly now, so I apologise for wasting your time. Thanks for taking the time to investigate.
